storagenode/storagenodedb/database.go

```diff
@@ -116,17 +237,421 @@ func (db *DB) Close() error {
 		db.ndb.Close(),
 		db.adb.Close(),
 
-		db.pieces.Close(),
-		db.info.Close(),
+		db.versions.Close(),
+		db.v0PieceInfo.Close(),
+		db.bandwidth.Close(),
+		db.console.Close(),
+		db.orders.Close(),
+		db.pieceExpirationDB.Close(),
+		db.pieceSpaceUsedDB.Close(),
+		db.reputation.Close(),
+		db.storageUsage.Close(),
+		db.usedserials.Close(),
```

Comment:
maybe nit/out of scope of this pr but why doesn't `usedserials` use camelcase?

Reply:
Done!